lib/srv/usermgmt.go

```diff
@@ -133,7 +133,7 @@ type HostUserManagement struct {
 
 var _ HostUsers = &HostUserManagement{}
 
-var sudoersSanitizationMatcher = regexp.MustCompile(`\.|~`)
+var sudoersSanitizationMatcher = regexp.MustCompile(`[\.~\/]`)
```

Comment:
Can you please add a comment here with reference to sudoers man page that explains that these characters are restricted.